src/status_im2/contexts/wallet/accounts/view.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.wallet.accounts.view
```

Comment:
Apologies, I checked the Figma now. The list of accounts is displayed in the home screen/tab. We will display the detailed information of a selected (tapped) account on this screen.

Should we drop the plural form (`accounts`) in ns and rename it to `status-im2.contexts.wallet.account.view` and update the screen name in navigation? This goes the same for `collectibles` too.  It's similar to the `chat` screen. WDYT?

Reply:
@smohamedjavid, yes I think you are right. I'll adjust this 👍